schemas/gazette-notice-schema.json

```diff
@@ -0,0 +1,171 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Notice",
+  "description": "A gazette notice",
+  "type": "object",
+  "properties": {
+    "issue": {
+      "description": "The issue in which the notice is published",
+      "$ref": "#/definitions/issue"
+    },
+    "identifier": {
+      "description": "The identifier of the notice",
+      "type": "string"
+    },
+    "title": {
+      "description": "The title of the notice",
+      "type": "string"
+    },
+    "description": {
+      "description": "A description of the notice",
+      "type": "string"
+    },
+    "body": {
+      "description": "The content of the notice",
```

Comment:
What form is this body in? I'm guessing not necessarily the same as the media_type of the URL, as that might be PDF. Assuming we have some that are html, plain text, "simple text" or even XML think we should record if possible. Will help parsing and rendering